slither/core/solidity_types/elementary_type.py

```diff
@@ -189,7 +189,7 @@ def size(self) -> Optional[int]:
         if t == "address":
             return int(160)
         if t.startswith("bytes"):
-            return int(t[len("bytes") :])
+            return int(t[len("bytes") :]) * 8
```

Comment:
Will this just result in 0 for `bytes`? Shouldn't it return `None` for that?

Reply:
Thanks for alerting me of this. We will need to check that the type is not exactly equal to `bytes` and then it will return `None`. 

```
if t.startswith("bytes") and t != "bytes":
```

Currently, Slither parses `bytes` (dynamic array in storage) as an instance of `ElementaryType`. Should it instead be an `ArrayType` like `uint8[]`?

Reply:
I'm just a person reporting bugs on this repository, so I can't really help whether this is good/bad/desired/undesired, but it did look unclear to me. I do see that `storage_size` has its own pre-checks though.